components/infobox/wikis/mobilelegends/infobox_league.lua

```diff
@@ -79,6 +79,7 @@ end
 
 function League:addToLpdb(lpdbData, args)
 	lpdbData.participantsnumber = args.player_number or args.team_number
+	lpdbData.publishertier =  _args['moonton-sponsored']
```

Comment:
considering that on line 61 we explicitly check: `_args['moonton-sponsored'] == 'true'`, maybe we should use
```suggestion
	lpdbData.publishertier =  _args['moonton-sponsored'] == 'true' and 'true' or nil
```